src/components/Navigation/components/Item/Item.tsx

```diff
@@ -226,7 +252,10 @@ export default class Item extends React.Component<Props, State> {
           onClick={this.getClickHandler(onClick)}
         >
           {iconMarkup}
-          <span className={styles.Text}>{label}</span>
+          <span className={styles.Text}>
+            {label}
+            {indicatorMarkup}
```

Comment:
For the UnstyledLink code path here, the indicator markup is a child of the `.Text` span. For the button path it’s the next sibling. Can they not be done the same way?

Reply:
Good catch. They should be the same. I moved this content up to a variable defined above so they always stay in sync.